Sources/BuildSystemIntegration/CompilerCommandLineOption.swift

```diff
@@ -39,16 +43,28 @@ package struct CompilerCommandLineOption {
   /// The name of the option, without any preceeding `-` or `--`.
   private let name: String
 
+  /// The name of the argument that prefixes this flag (eg. `-Xfrontend`/`-Xclang`).
+  private let frontendName: String?
```

Comment:
The documentation doesn’t entirely match the behavior. The implementation expects `Xfrontend` while the documentation mentions `-Xfrontend`.

Reply:
It was meant to be an example of a flag 😅 But I'll change given I forgot to update cmakelists :(